data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,35 @@ mission "Hai Reveal [A08] Symbol Identification"
 		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
 		
 		conversation
-			`After landing, you search up the way to the adjunct research library at the Navy Intelligence Observatory in Moscow. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours. For someone of your background there are parts of this trip that seem dizzyingly unreal. As well traveled as you now are, actually navigating the urban environments of <origin> away from any particular spaceport is an experience not quite like any other. Once you reach the building in question you take a moment at the reception desk to interpret 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your other business about the symbol.`
+			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

Comment:
```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can travel by shuttle, which is slower but offers the chance to see some of the sights of Earth. Either way, you'll have plenty of time before Batu arrives.`
```

Reply:
The long route also involves a train and some walking. "By shuttle" implies that the shuttle takes you all the way, which is actually the other option.

Reply:
Okay, but the information desk will tell you to take some flights to Nizhny Novgorod first. I'm pretty sure that's not mentioned until you get there. 
```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can travel by shuttle to Nizhny and take the train to Moscow from there. It'll be slower, but you'll have the chance to see some of the sights of Earth. Either way, you'll have plenty of time before Batu arrives.`
```

Reply:
Nevermind, it wouldn't work with the next bit.